pb/unixfs.pb.go

```diff
@@ -18,7 +18,7 @@ var _ = math.Inf
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.GoGoProtoPackageIsVersion2 // please upgrade the proto package
+const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package
```

Comment:
To maintain compatibility, this should not change

Reply:
I did not change intentionally. Probably protoc simply compile in version 3 by default. 
I will correct this one along with the above gogo part.

Reply:
sure